.github/workflows/new_issues.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           project-url: https://github.com/orgs/WordPress/projects/75
           github-token: ${{ secrets.ACCESS_TOKEN }}
           # Exclude the issues with the following labels
-          labeled: project
+          labeled: "🧭 project: thread, 🧭 project: proposal, 🧭 project: implementation plan"
```

Comment:
```suggestion
          labeled: "🧭 project: thread"
```

Reply:
The proposals and implementations plans are issues that require PRs and reviews, so they shouldn't be ignored.